src/librustc_trans/trans/base.rs

```diff
@@ -745,13 +745,22 @@ pub fn invoke<'blk, 'tcx>(bcx: Block<'blk, 'tcx>,
     }
 }
 
+/// Returns whether this session's target will use SEH-based unwinding.
+///
+/// This is only true for MSVC targets, and even then the 64-bit MSVC target
+/// currently uses SEH-ish unwinding with DWARF info tables to the side (same as
+/// 64-bit MinGW) instead of "full SEH".
+pub fn wants_seh(sess: &Session) -> bool {
```

Comment:
I find this nomenclature confusing: the format of LSDA is not a part of the SEH spec, so both -gnu and -msvc are "full SEH".  I think this should be called `wants_msvc_lsda`, or at least `wants_msvc_seh`.